### ISSUE TITLE
quasar dev fails

### ISSUE THREAD

**Issue by paulvanbladel** [thibautguedou3/quasar-storybook-typescript#1]
Very nice storybook integration, unfortunately it compromises the general functioning of the app

quasar dev -->
```
These dependencies were not found:

* core-js/modules/es6.array.find in ./node_modules/quasar/src/polyfills.js
* core-js/modules/es6.array.iterator in ./node_modules/quasar/src/polyfills.js, ./node_modules/quasar/src/lang.js and 11 others
* core-js/modules/es6.function.name in ./node_modules/quasar/src/icon-set.js, ./node_modules/quasar/src/install.js and 1 other
* core-js/modules/es6.number.constructor in ./node_modules/quasar/src/polyfills.js, ./node_modules/quasar/src/components/btn/QBtn.js and 9 others
* core-js/modules/es6.number.is-integer in ./node_modules/quasar/src/polyfills.js
* core-js/modules/es6.object.assign in ./node_modules/quasar/src/polyfills.js, ./node_modules/quasar/src/ssr-update.js and 4 others
* core-js/modules/es6.object.keys in ./node_modules/quasar/src/lang.js, ./node_modules/quasar/src/ssr-update.js and 11 others
* core-js/modules/es6.object.to-string in ./node_modules/quasar/src/install.js, ./node_modules/quasar/src/lang.js and 12 others
* core-js/modules/es6.regexp.replace in ./.quasar/client-entry.js, ./node_modules/quasar/src/components/btn/QBtn.js and 2 others
* core-js/modules/es6.regexp.split in ./node_modules/quasar/lang/en-us.js, ./node_modules/quasar/src/components/layout/QLayout.js and 1 other
* core-js/modules/es6.regexp.to-string in ./node_modules/quasar/src/polyfills.js, ./node_modules/quasar/src/utils/colors.js
* core-js/modules/es6.string.ends-with in ./node_modules/quasar/src/polyfills.js
* core-js/modules/es6.string.fixed in ./node_modules/quasar/src/components/layout/QHeader.js, ./node_modules/quasar/src/components/layout/QDrawer.js
* core-js/modules/es6.string.includes in ./node_modules/quasar/src/ssr-update.js, ./node_modules/quasar/src/polyfills.js and 4 others
* core-js/modules/es6.string.starts-with in ./node_modules/quasar/src/polyfills.js, ./node_modules/quasar/src/components/icon/QIcon.js and 1 other
* core-js/modules/es6.symbol in ./node_modules/quasar/src/ssr-update.js, ./node_modules/quasar/src/components/btn/QBtn.js and 4 others
* core-js/modules/es7.array.includes in ./node_modules/quasar/src/ssr-update.js, ./node_modules/quasar/src/polyfills.js and 4 others
* core-js/modules/es7.object.get-own-property-descriptors in ./node_modules/quasar/src/ssr-update.js, ./node_modules/quasar/src/components/btn/QBtn.js and 4 others
* core-js/modules/web.dom.iterable in ./node_modules/quasar/src/polyfills.js, ./node_modules/quasar/src/ssr-update.js and 11 others

To install them, you can run: npm install --save core-js/modules/es6.array.find core-js/modules/es6.array.iterator core-js/modules/es6.function.name core-js/modules/es6.number.constructor core-js/modules/es6.number.is-integer core-js/modules/es6.object.assign core-js/modules/es6.object.keys core-js/modules/es6.object.to-string core-js/modules/es6.regexp.replace core-js/modules/es6.regexp.split core-js/modules/es6.regexp.to-string core-js/modules/es6.string.ends-with core-js/modules/es6.string.fixed core-js/modules/es6.string.includes core-js/modules/es6.string.starts-with core-js/modules/es6.symbol core-js/modules/es7.array.includes core-js/modules/es7.object.get-own-property-descriptors core-js/modules/web.dom.iterable
```

**Comment by thibautguedou3** [comment]
Hi @paulvanbladel ! When I launch `quasar dev` with the typescript set up, everything goes as expected, I do not have your error. Could you show me the content of your quasar.conf file so that I can look it up?

**Comment by paulvanbladel** [comment]
@thibautguedou3 Hi Thibaut. Thanks for the prompt reply.
The quasar.conf is that from your repo.
Are you using yarn as well. I just did a completely fresh git clone from your repo and did yarn to install the deps. I still get the eror as above.
Could you maybe delete your node_modules folder and the yarn.lock file and then install the deps again? That should normally lead to the same problem.
This is my quasar info:
```
Operating System - Windows_NT(10.0.17763) - win32/x64
NodeJs - 8.12.0

Global packages
  NPM - 6.4.1
  yarn - 1.13.0
  @quasar/cli - 1.0.0
  cordova - Not installed

Important local packages
  quasar - 1.2.4 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 1.2.2 -- Quasar Framework local CLI
  @quasar/extras - 1.3.2 -- Quasar Framework fonts, icons and animations
  vue - 2.6.10 -- Reactive, component-oriented view layer for modern web interfaces.
  vue-router - 3.1.3 -- Official router for Vue.js 2
  vuex - 3.1.1 -- state management for Vue.js
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed
  @babel/core - 7.6.4 -- Babel compiler core.
  webpack - 4.41.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple
bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 3.8.2 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - 4.3.1 -- A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.
  register-service-worker - 1.6.2 -- Script for registering service worker, with hooks

Quasar App Extensions
  *None installed*
````

**Comment by thibautguedou3** [comment]
No I'm using npm, I have tried to delete my node_modules folder and my package-lock.json file and then launch npm install and everything works fine.
I tried with yarn and I had the same problem as you described. To fixed it, I ran `yarn add core-js@2.6.5` as suggested in [this issue](https://github.com/brunocodutra/webapp-webpack-plugin/issues/72#issuecomment-479233871) . It's a bit strange that you have to do this only with yarn but I hope it will fix your problem :)

**Comment by paulvanbladel** [comment]
@thibautguedou3 Brilliant, that did the job. Allow me to thank you for your continuous help.
